packages/dx-react-scheduler/docs/guides/appointment-tooltip.md

```diff
@@ -0,0 +1,31 @@
+# React Scheduler - Appointment Tooltip
+
+The React Scheduler component's `AppointmentTooltip` plugin displays compact appointment data.
```

Comment:
I think it's better to replace 'compact' with 'detailed' and to mention that an appointment tooltip can contain some action buttons

Reply:
`Detailed` means `additional information`. `Compact` means `little information`. I think we should keep this word.

Reply:
An appointment tooltip as a concept is often a panel with summarized info and actions - a date/time, description, resource name, and so on. From the other hand, appointments are often kept as minimalistic as possible to not distract an attention, e.g. in month view we render only a caption not date/time. So we can say that a tooltip displays extended/additional information :)

Reply:
Okay, you are right)